package.json

```diff
@@ -32,7 +32,8 @@
 
   "scripts": {
     "prepublish": "make -j8",
-    "test": "./node_modules/.bin/mocha",
-    "watch": "./node_modules/.bin/watchman src 'npm run prepublish'"
+    "start": "./bin/fxos-device-service",
+    "test": "mocha",
```

Comment:
Since npm adds `node_modules/.bin` to the `$PATH` of npm scripts, this will use the local mocha, which is installed in `devDependencies`.